lib/smart_todo/dispatchers/base.rb

```diff
@@ -91,6 +91,15 @@ def unexisting_user(assignee)
       def existing_user
         "Hello :wave:,"
       end
+
+      def repo
+        repo = @options[:repo]
+        return unless repo
+
+        unless repo.empty?
+          " in repository `#{repo}`"
```

Comment:
up to you but I think mentioning repository is redundant, to me it would be clearly implied if I see a message like "TODO in the `models/order.rb` in `shop/world`
```suggestion
          " in `#{repo}`"
```

*Unless `repo` is just the repo name (like `world`) and it doesn't include org